Directory.Build.props

```diff
@@ -0,0 +1,43 @@
+<Project>
+  <PropertyGroup>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+  <PropertyGroup>
+    <Configurations>Debug;Release;Debug_ReadOnly;Release_ReadOnly</Configurations>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)\cecil.snk</AssemblyOriginatorKeyFile>
+    <DefineConstants Condition="'$(TargetFramework)'=='net35'">$(DefineConstants);NET_3_5;</DefineConstants>
+    <DefineConstants Condition="'$(TargetFramework)'=='net40'">$(DefineConstants);NET_4_0;</DefineConstants>
+    <DefineConstants Condition="'$(TargetFramework)'=='netstandard2.0' OR '$(TargetFramework)'=='netcoreapp2.1'">$(DefineConstants);NET_CORE;</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition=" $(Configuration.Contains('ReadOnly')) ">
+    <DefineConstants>$(DefineConstants);READ_ONLY</DefineConstants>
+  </PropertyGroup>
+   <PropertyGroup Condition="'$(TargetFramework)' == 'net35' ">
+    <DefineConstants>$(DefineConstants);LEGACY</DefineConstants>
```

Comment:
Is the `LEGACY` define used? A quick search in the code doesn't find anything.

Reply:
I think I put this at a moment where I was using it but it's no more needed. Could be removed.